src/Microsoft.SqlTools.Hosting/Utility/GeneralRequestDetails.cs

```diff
@@ -14,7 +14,7 @@ public class GeneralRequestDetails
     {
         public GeneralRequestDetails()
         {
-            Options = new Dictionary<string, object>();
+            Options = new Dictionary<string, object>(new OptionsKeyComparer());
         }
```

Comment:
Could we use StringComparer.InvariantCultureIgnoreCase instead of defining our own?

Reply:
yes absolutely... will change